composer.json

```diff
@@ -33,6 +33,7 @@
     "require": {
         "php": ">=5.5",
         "ext-dom": "*",
+        "ext-json": "*",
```

Comment:
ext-json is not required, but can be suggested => not required if you don't use the report

Reply:
After more than 18 months, I finally removed it :laughing: